addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -490,43 +490,43 @@
             model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fp_not_subject_tai"/>
             <field name="tax_src_id" ref="account_tax_template_s_iva4b"/>
-            <field name="tax_dest_id" ref="account_tax_template_s_iva_ns_b"/>
+            <field name="tax_dest_id" ref="account_tax_template_s_iva0_e"/>
         </record>
         <record id="fptt_not_subject_tai_ventas_4s"
             model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fp_not_subject_tai"/>
             <field name="tax_src_id" ref="account_tax_template_s_iva4s"/>
-            <field name="tax_dest_id" ref="account_tax_template_s_iva_ns"/>
+            <field name="tax_dest_id" ref="account_tax_template_s_iva0_e"/>
```

Comment:
you are right with this one @jco-odoo We need to keep it as account_tax_template_s_iva_ns